CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ else()
   target_sources(ifc-reader PRIVATE src/sha256.cxx)
 endif()
 target_link_libraries(ifc-reader PUBLIC Microsoft.GSL::GSL)
-target_compile_features(ifc-reader PUBLIC cxx_std_20)
+target_compile_features(ifc-reader PUBLIC cxx_std_23)
```

Comment:
`cxx_std_23` is only supported by CMake 3.20+ https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html#high-level-meta-features-indicating-c-standard-support

Currently, the project promises to work with CMake 3.14+, which it does not anymore.

Reply:
> Currently, the project promises to work with CMake 3.14+, which it does not anymore.

Hmm, where did me promise that?

Reply:
`cmake_minimum_required` is a promise from the project author(s) to consumers that the project uses features of at most that version. Currently that declared to be CMake 3.14, which does not know about C++23.

Reply:
Oh, that should also have been updated to 3.27. Sorry, I completely missed that.